source/PvL_07.rst

```diff
@@ -224,42 +197,13 @@ omitting any lines that begin with ``#``:
        :linenos:
         
         def filter(oldfile, newfile):
-            infile = open(oldfile, "r")
-            outfile = open(newfile, "w")
-            while True:
-                text = infile.readline()
-                if len(text) == 0: 
-                    break
-                if text[0] == "#":
-                    continue
-                   
-                # Put any more processing logic here
-                outfile.write(text)
-                
-            infile.close()
-            outfile.close()
-
-In this case, we explicitly open and close the files, instead of using the with statement used above. 
-This is because we're working with two files, and this way, they code is much neater.
-
-The ``continue`` statement at line 9 skips over the remaining lines in
-the current iteration of the loop, but the loop will still iterate.  This
-style looks a bit contrived here, but it is often useful to say *"get the
-lines we're not concerned with out of the way early, so that we have
-cleaner more focused logic in the meaty part of the loop that might be
-written around line 11."* 
```

Comment:
I think something like this should be here. It's does need to be worded better though.